SConstruct

```diff
@@ -679,7 +679,7 @@ libraries = {
 
 	"GafferScene" : {
 		"envAppends" : {
-			"LIBS" : [ "Gaffer", "Iex$OPENEXR_LIB_SUFFIX", "IECoreGL$CORTEX_LIB_SUFFIX", "IECoreImage$CORTEX_LIB_SUFFIX",  "IECoreScene$CORTEX_LIB_SUFFIX", "GafferImage", "GafferDispatch" ],
+			"LIBS" : [ "Gaffer", "Iex$OPENEXR_LIB_SUFFIX", "IECoreGL$CORTEX_LIB_SUFFIX", "IECoreImage$CORTEX_LIB_SUFFIX",  "IECoreScene$CORTEX_LIB_SUFFIX", "GafferImage", "GafferDispatch", "Half" ],
```

Comment:
> This seems to have been getting included implicitly on Linux, but failing on OS X.

Bit of background :

By default, the Linux linker doesn't worry about unresolved symbols - it just assumes they'll be available at runtime. So as long as GafferScene is loaded into an executable that already happens to have loaded libHalf, all is well. By default, the Mac linker isn't so lenient - it insists on being able to resolve everything at link time.

This is actually catching us out fairly regularly, so I think perhaps we should add whatever flag make Linux behave the same way as Mac. I think I even have an email from Don somewhere telling me what the right incantation is...